### PR TITLE
Improve Cheat Sheets

### PR DESCRIPTION
**Pull request by vikashvverma**
IA Pages :

https://duck.co/ia/view/es6_cheat_sheet 
https://duck.co/ia/view/emmet_cheat_sheet
https://duck.co/ia/view/emberjs_cheat_sheet
https://duck.co/ia/view/emberjs_cheat_sheet
https://duck.co/ia/view/emacs_idris_mode_cheat_sheet
https://duck.co/ia/view/docker_cheat_sheet
https://duck.co/ia/view/default_port_numbers_cheat_sheet
https://duck.co/ia/view/coffeescript_cheat_sheet
https://duck.co/ia/view/chess_cheat_sheet
https://duck.co/ia/view/bash_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@vikashvverma thanks a lot for doing this!

File: share/goodie/cheat_sheets/json/coffeescript.json [hunk 1]
```diff
@@ -125,6 +125,9 @@
             "val": "find element in array (No JS equivalent)",
             "key": "in"
         }, {
+            "val": "Eistential operator: returns true unless 'null' or 'undefined'",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/coffeescript.json]: Looks like a typo: `Existential`

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/coffeescript.json]: Fixed it!

**Comment by vikashvverma** [comment]
@moollaza Can you review this. I think it's almost done. Rest cheat sheets are new IA and can remain in separate PRs.

**Comment by GuiltyDolphin** [comment]
@vikashvverma You've got merge conflicts! And it looks like you've got some other commits (from others) buried in there? Lets get this branch sorted so we can continue with the review.

**Comment by vikashvverma** [comment]
@GuiltyDolphin Have a look at this PR https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2746

**Pull request closed without merging**